proxy/src/transport/tls/config.rs

```diff
@@ -271,8 +279,8 @@ pub fn watch_for_config_changes(settings: Option<&CommonSettings>)
                 Ok((client_store, server_store))
             })
         .then(|_| {
-            trace!("forwarding to server config watch finished.");
-            Ok(())
+            error!("forwarding to tls config watches finished.");
+            Err(())
```

Comment:
Furthermore, unless we expect this stream to end, I think we should consider it to be an error.

Reply:
I agree, this stream should be infinite.